Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:wheezy
+FROM ubuntu:14.04
```

Comment:
Why have you reverted this to ubuntu:14.04?

We build on debian for various dependency reasons (libc version, etc).